File: 215.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        // 最直接的想法，直接sort取 nums[size - k]
        sort(nums.begin(), nums.end());
        int size = nums.size();
        return nums[size - k];
    }
};

File: 215.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        headp = []
        for num in nums:
            heapq.heappush(heap, num)

            if(len(heap) > k):
                headpq.popback(headp)
        return headp[0]
